[2022-03-17 03:22:11,758] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-08-02T06:00:00+00:00 [queued]>
[2022-03-17 03:22:11,809] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-08-02T06:00:00+00:00 [queued]>
[2022-03-17 03:22:11,811] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:22:11,812] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:22:11,813] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:22:11,842] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-08-02T06:00:00+00:00
[2022-03-17 03:22:11,848] {standard_task_runner.py:52} INFO - Started process 1014 to run task
[2022-03-17 03:22:11,858] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-08-02T06:00:00+00:00', '--job-id', '22', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmpnpqz_5ok', '--error-file', '/tmp/tmp6vta7t3r']
[2022-03-17 03:22:11,866] {standard_task_runner.py:77} INFO - Job 22: Subtask ingest
[2022-03-17 03:22:11,993] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-08-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:22:12,115] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-08-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-08-02T06:00:00+00:00
[2022-03-17 03:22:12,118] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:22:12,122] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:22:12,136] {bash.py:169} INFO - Output:
[2022-03-17 03:22:12,212] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:22:12,213] {bash.py:173} INFO - dags
[2022-03-17 03:22:12,217] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:22:12,218] {bash.py:173} INFO - logs
[2022-03-17 03:22:12,220] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:22:12,222] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:22:12,224] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:22:12,226] {bash.py:173} INFO - output_2019-04.csv
[2022-03-17 03:22:12,229] {bash.py:173} INFO - output_2019-05.csv
[2022-03-17 03:22:12,234] {bash.py:173} INFO - output_2019-06.csv
[2022-03-17 03:22:12,239] {bash.py:173} INFO - output_2019-07.csv
[2022-03-17 03:22:12,240] {bash.py:173} INFO - output_2019-08.csv
[2022-03-17 03:22:12,241] {bash.py:173} INFO - output_2019-09.csv
[2022-03-17 03:22:12,243] {bash.py:173} INFO - plugins
[2022-03-17 03:22:12,246] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:22:12,247] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:22:12,249] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:22:12,324] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190802T060000, start_date=20220317T032211, end_date=20220317T032212
[2022-03-17 03:22:12,392] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:22:12,432] {local_task_job.py:146} INFO - Task exited with return code 0
